docs/Server-configuration.md

```diff
@@ -93,3 +93,66 @@ IMPOSTOR_Server__ListenPort=22023
 IMPOSTOR_AntiCheat__Enabled=true
 IMPOSTOR_AntiCheat__BanIpFromGame=true
 ```
+
+# systemd
+
+Linux users will likely want to create a systemd service definition to manage running the Impostor application.  Benefits to using a systemd service include the following:
+
+- The server process can automatically restart if the process dies
+- Impostor can be set to start on boot (handy if you do system updates regularly)
+- You can use systemd targets to ensure the process doesn't start until the system dependencies are met
+
+Using your favorite text editor, create the following service file:
+
+```
+sudo vim /etc/systemd/system/impostor.service
+```
+
+Modify the following template to your needs and populate the service file.
+
+```
+[Unit]
+Description=Impostor private Among Us server - https://github.com/Impostor/Impostor
```

Comment:
I'm looking at my own unit file (probably should've upstreamed that sooner but oops) and I also have two exta lines here:

```
After=syslog.target
After=network.target
```

This seems to be orthagonal to WantedBy, should I add these lines here or are they sort of obsolete?

Reply:
WantedBy=multiuser.target adds the multiuser target as a dependency.  I think it's a slightly cleaner way of doing it.  Ultimately your way would also likely work.

Reply:
It's not `After` or `WantedBy`, you want both:
- After specifies this unit must start when a few other units have also started
- WantedBy gives a deadline: this unit must start for the system to consider itself to be fully started.